sdk/lib/cmlib/hiveinit.c

```diff
@@ -345,8 +476,16 @@ HvpInitializeFlatHive(
     PHHIVE Hive,
     PHBASE_BLOCK ChunkBase)
 {
-    if (!HvpVerifyHiveHeader(ChunkBase))
+    if (!HvpVerifyHiveHeader(ChunkBase, HFILE_TYPE_PRIMARY))
```

Comment:
Technically, flat hives don't need the log restoration, contrary to the in-memory hives (`HINIT_MEMORY`, loaded by `HvpInitializeMemoryHive()`).